wavefront/resource_alert.go

```diff
@@ -76,7 +76,7 @@ func resourceAlert() *schema.Resource {
 			"severity": {
 				Type:             schema.TypeString,
 				Optional:         true,
-				DiffSuppressFunc: suppressCase,
+				DiffSuppressFunc: ignoreChanges,
```

Comment:
The severity field appears to be settable via both the users config and being returned from Wavefront. Perhaps instead of a DiffSuppressFunc, should it be both Optional and Computed being true?

Reply:
I'll test it without DiffSupressFunc and Computed = true to see if the alert firing still triggers a change in "terraform plan"

Reply:
Hi Mark, what you suggested seems to work.  I've updated the PR accordingly.